src/Form/Type/NewOrderType.php

```diff
@@ -75,6 +75,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                         'choices' => $event->getData()->getChannel()->getLocales(),
                         'empty_data' => $event->getData()->getLocaleCode(),
                     ])
+                    ->add('currencyCode', CurrencyCodeChoiceType::class, [
+                        'label' => false,
+                        'placeholder' => 'sylius.ui.currency',
+                        'choices' => $event->getData()->getChannel()->getCurrencies(),
```

Comment:
We should at least try to reduce the number of dzidas on each line. Can you extract `$event->getData()` and add typehint?